src/Symfony/Component/Routing/Tests/Loader/AnnotationClassLoaderTest.php

```diff
@@ -69,187 +77,189 @@ public function testSupportsChecksTypeIfSpecified()
-        $route = $routeCollection->get($classRouteData['name']);
-
-        $this->assertSame($classRouteData['path'], $route->getPath(), '->load preserves class route path');
-        $this->assertEquals(array_merge($classRouteData['schemes'], $classRouteData['schemes']), $route->getSchemes(), '->load preserves class route schemes');
-        $this->assertEquals(array_merge($classRouteData['methods'], $classRouteData['methods']), $route->getMethods(), '->load preserves class route methods');
-    }
-
-    public function testInvokableClassWithMethodRouteLoad()
-    {
-        $classRouteData = array(
-            'name' => 'route1',
-            'path' => '/prefix',
-            'schemes' => array('https'),
-            'methods' => array('GET'),
-        );
-
-        $methodRouteData = array(
-            'name' => 'route2',
-            'path' => '/path',
-            'schemes' => array('http'),
-            'methods' => array('POST', 'PUT'),
-        );
-
-        $this->reader
-            ->expects($this->once())
-            ->method('getClassAnnotation')
-            ->will($this->returnValue($this->getAnnotatedRoute($classRouteData)))
-        ;
-        $this->reader
-            ->expects($this->once())
-            ->method('getMethodAnnotations')
-            ->will($this->returnValue(array($this->getAnnotatedRoute($methodRouteData))))
-        ;
-
-        $routeCollection = $this->loader->load('Symfony\Component\Routing\Tests\Fixtures\AnnotatedClasses\BazClass');
-        $route = $routeCollection->get($classRouteData['name']);
-
-        $this->assertNull($route, '->load ignores class route');
-
-        $route = $routeCollection->get($classRouteData['name'].$methodRouteData['name']);
-
-        $this->assertSame($classRouteData['path'].$methodRouteData['path'], $route->getPath(), '->load concatenates class and method route path');
-        $this->assertEquals(array_merge($classRouteData['schemes'], $methodRouteData['schemes']), $route->getSchemes(), '->load merges class and method route schemes');
-        $this->assertEquals(array_merge($classRouteData['methods'], $methodRouteData['methods']), $route->getMethods(), '->load merges class and method route methods');
-    }
-
-    private function getAnnotatedRoute($data)
-    {
-        return new Route($data);
+    /**
+     * @test
+     */
+    public function simple_path_routes()
+    {
+        $routes = $this->loader->load(ActionPathController::class);
+        $this->assertCount(1, $routes);
+        $this->assertEquals('/path', $routes->get('action')->getPath());
+    }
+
+    /**
+     * @test
+     */
+    public function invokable_controller_loading()
+    {
+        $routes = $this->loader->load(InvokableController::class);
+        $this->assertCount(1, $routes);
+        $this->assertEquals('/here', $routes->get('lol')->getPath());
+    }
+
+    /**
+     * @test
```

Comment:
Please don't change all existing tests, especially when your new code does not follow the Symfony conventions. Changing existing tests makes it much harder for reviewers (I would now have to review all tests to be sure that the new ones are equivalent to the old ones, except I won't do it and I will wait for a revert instead).